### PR TITLE
Update Azure Monitor exporter for Python

### PR DESCRIPTION
**Pull request by reyang**

### CODE REVIEW

**Comment by reyang** [comment]
FYI @lzchen

**Comment by reyang** [comment]
#358

File: content/exporters/supported-exporters/Python/AzureMonitor.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  [here](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource).
+* Put the instrumentation key in `APPINSIGHTS_INSTRUMENTATIONKEY` environment variable.
+* Create the exporters in code.
+
+{{<highlight python>}}
+from opencensus.ext.azure.trace_exporter import AzureExporter
+from opencensus.trace.samplers import ProbabilitySampler
+from opencensus.trace.tracer import Tracer
+
+tracer = Tracer(exporter=AzureExporter(), sampler=ProbabilitySampler(1.0))
```
Comment (songy23) [inline content/exporters/supported-exporters/Python/AzureMonitor.md]: Consider adding a comment that 100% sampling is only for demo purpose.

Comment (reyang) [reply content/exporters/supported-exporters/Python/AzureMonitor.md]: Got it.

Comment (reyang) [reply content/exporters/supported-exporters/Python/AzureMonitor.md]: Added.

**Pull request merged**